lib/cap_authors_poller.rb

```diff
@@ -126,6 +126,11 @@ def queue_author_for_harvest(author, skip_message)
   end
 
   def update_existing_contributions(author, incoming_authorships)
+    # inject author identity into the authorship data
+    incoming_authorships = incoming_authorships.dup.map do |contrib|
+      contrib[:cap_profile_id] = author.cap_profile_id
+      contrib
+    end
```

Comment:
Note also that
- https://github.com/sul-dlss/sul_pub/blob/master/lib/cap_authors_poller.rb#L130
Could be modified to only validate the fields by instead just calling
- https://github.com/sul-dlss/sul_pub/blob/master/app/models/contribution.rb#L32